scripts/src/owners/owners_file.py

```diff
@@ -33,6 +33,15 @@ def get_vendor(owner_data):
     return vendor
 
 
+def get_vendor_label(owner_data):
+    vendor = ""
+    try:
+        vendor = owner_data["vendor"]["label"]
+    except Exception:
```

Comment:
I think this can be rewritten to be more functional for us as a whole.

With that said, I want to maintain this change to be as minimal as possible for the purposes of reducing modified code paths. TL;DR I'd like to improve this in the future, unrelated to this bugfix.

Reply:
Opened https://github.com/openshift-helm-charts/development/issues/317 to track this future work.